metastats-infrastructure/src/main/groovy/life/qbic/metastats/filter/JsonValidator.groovy

```diff
@@ -24,7 +25,7 @@ class JsonValidator implements SchemaValidator {
     //TODO use model schema from external resource like github
```

Comment:
Is this still relevant?

Reply:
Yes, we want to once move the schema to an external source but this will be done in a later version of the software. If it bothers you I can remove it (can be added to issue #7 )

Reply:
I'd suggest removing it for the release version and adding it to the issue as you suggested. But we're really in the Nitpick territory on my part here :)